### PR TITLE
Variant prices in populatedb

### PR DESCRIPTION
**Pull request by szewczykmira**
Force random prices assignment to make real-life sense. Works based on assumption that cheaper variant will be closer to beginning of the list, i.e. list of variant attributes is sorted on price - cheapest first.

### CODE REVIEW

**Comment by patrys** [comment]
I think the most realistic scenario is that variants with price overrides are _more expensive_ than the base product price.

File: saleor/core/utils/random_data.py [hunk 1]
```diff
@@ -184,14 +184,14 @@ def get_variant_combinations(product):
...
-    return [{str(attr_value.attribute.pk): str(attr_value.pk)}
-            for combination in all_combinations
-            for attr_value in combination]
+    return [{str(attr_value.attribute.pk): str(attr_value.pk)
+            for attr_value in combination}
+            for combination in all_combinations]
 
 
-def get_price_override(schema):
+def get_price_override(schema, n):
```
Comment (mociepka) [inline saleor/core/utils/random_data.py]: Could we use more verbose name than `n`?

File: saleor/core/utils/random_data.py [hunk 2]
```diff
@@ -210,10 +210,14 @@ def create_products_by_class(product_class, schema,
...
                 product, random.randrange(1, 5), class_placeholders)
         variant_combinations = get_variant_combinations(product)
+        prices = get_price_override(schema, len(variant_combinations))
         for i, attr_combination in enumerate(variant_combinations, 1337):
             sku = '%s-%s' % (product.pk, i)
-            create_variant(product, attributes=attr_combination,
-                           price_override=get_price_override(schema), sku=sku)
+            kwargs = {'attributes': attr_combination, 'sku': sku}
+            if schema.get('different_variant_prices'):
+                kwargs['price_override'] = product.price + next(prices)
```
Comment (mociepka) [inline saleor/core/utils/random_data.py]: Using `zip` and having it in for loop variable will be more readable.

File: saleor/core/utils/random_data.py [hunk 3]
```diff
@@ -210,10 +210,14 @@ def create_products_by_class(product_class, schema,
             create_product_images(
                 product, random.randrange(1, 5), class_placeholders)
         variant_combinations = get_variant_combinations(product)
+        prices = get_price_override(schema, len(variant_combinations))
         for i, attr_combination in enumerate(variant_combinations, 1337):
```
Comment (mociepka) [inline saleor/core/utils/random_data.py]: Why `1337`? Maybe it is too much but in my opinion digits and bools should be passes named:
```python
enumerate(variant_combinations, start=1337)
```

It is way more readable.

Comment (patrys) [inline saleor/core/utils/random_data.py]: `enumerate` is a builtin. Should we also provide keyword args to `range`?

**Pull request merged**